samples/widgets/clustering/src/README.md

```diff
@@ -0,0 +1,97 @@
+# Clustering
+
+This sample demonstrates how to enable point clustering on a feature layer in a web map. 
+
+## How to use the sample
+Clone the [sample repo](https://github.com/esri/arcgis-experience-builder-sdk-resources) and copy this widget's folder (within `samples/widgets`) to the `client/your-extensions/widgets` folder of your Experience Builder installation.
+
+## How it works
+The property `clusterConfig` defines the cluster properties such as the `clusterRadius`, which determines each cluster's region for including features. 
+
+  ```javascript
+ clusterConfig = {
+  type: "cluster",
+  clusterRadius: "100px",
+  popupTemplate: {
+    content: "This cluster represents <b>{cluster_count}</b> points.",
+    fieldInfos: [{
+      fieldName: "cluster_count",
+      format: {
+        digitSeparator: true,
+        places: 0
+      }
+    }]
+  },
+    clusterMinSize: "24px",
+    clusterMaxSize: "60px",
+          labelingInfo: [{
+            deconflictionStrategy: "none",
+            labelExpressionInfo: {
+              expression: "Text($feature.cluster_count, '#,###')"
+            },
+         
+            labelPlacement: "center-center",
+          }]
+
+}
+
+```
+
+`onCheckBoxChange` function handles the logic for the `Checkbox` component and sets the state based on the value and passes this to the `clusterSwitch` function.
+```javascript
+ onCheckBoxChange = (e) => {
+  const valueState = e.target.value
+   this.setState({ 
+    clusterStatus: valueState 
+    },  () => {
+      this.clusterSwitch(this.state.clusterStatus);
+      })
+}
+
+```
+
+The `clusterSwitch` function controls the display of the cluster. If `this.state.clusterStatus` is true then enable clustering. Otherwise, turn off the clustering for the layer by setting it to `null`.    
```

Comment:
```suggestion
The `clusterSwitch` function controls the display of the cluster. If `this.state.clusterStatus` is true, then enable clustering. Otherwise, turn off the clustering for the layer by setting it to `null`.    
```